### PR TITLE
Implement admin dashboard

### PR DESCRIPTION
**Pull request by Plakhotenko**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 const App: FC = () => (
-  <I18nProvider>
-    <BrowserRouter>
-      <AuthProvider>
-        <Header />
-        <AppRoutes />
-      </AuthProvider>
-    </BrowserRouter>
-  </I18nProvider>
+  <LocalizationProvider dateAdapter={AdapterDateFns}>
```
Comment (Rammfall) [inline src/App.tsx:11]: It's just a comment

I did not see to provider for picker wrap all applications, but maybe it is normal

File: src/client.ts [hunk 2]
```diff
@@ -6,4 +6,16 @@ const client = axios.create({
   baseURL,
 });
 
+client.interceptors.request.use((config) => {
```
Comment (Rammfall) [inline src/client.ts]: maybe create folder (httpClient, client, api etc) and replace creating client here and create a folder with interceptors

File: src/components/FineForm/component.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { FC, useEffect, useState } from 'react';
```
Comment (Rammfall) [inline src/components/FineForm/component.tsx]: I want to suggest you use another flow for naming component
example
The component has the name FineForm
We create a folder FineForm, index.ts for exports/imports, and the component should have the name FineForm.tsx

Each component will have own name. If needs to replace logic in custom hooks only for this component, lets name FineForm.hooks.ts

File: src/components/FineForm/component.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  const validationSchema = useValidationSchema();
+
+  useEffect(() => {
+    admin
+      .users()
+      .then(({ data: users }) => setUsers(users))
+      .catch(() => setUsers([]));
+  }, []);
+
+  const onUserInputChange = debounce((event: any, value: string) => {
```
Comment (Rammfall) [inline src/components/FineForm/component.tsx]: lets use only one naming, `handle` for local handlers, `onSomething` for props

Comment (Rammfall) [reply src/components/FineForm/component.tsx]: Maybe set correct type to event params

File: src/services/admin.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import client from '../client';
+import { IUser } from '../models';
+
+export const users = (search?: string) =>
```
Comment (Rammfall) [inline src/services/admin.ts]: maybe getUsers

File: src/services/admin.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  client.get<IUser[]>('/users', { ...(search && { params: { search } }) });
+
+export const fine = (data: {
```
Comment (Rammfall) [inline src/services/admin.ts]: maybe createFine

**Review by Rammfall — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Plakhotenko) [reply src/App.tsx:11]: To me it is ok. It has been implemented according to the setup guide.

**Review by Rammfall — APPROVED** [review]

**Pull request merged**